org.eclipse.scanning.points/src/org/eclipse/scanning/points/RepeatedPointIterator.java

```diff
@@ -81,4 +67,21 @@ public int getRank() {
 		return 1;
 	}
 
+	// TODO: 2017-10-09 Matt D: These two fields and the get and set methods for countSleeps exist
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)